docs/UserGuide.md

```diff
@@ -206,25 +206,36 @@ _Details coming soon ..._
+|-----------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Create a new event**                        | `ecreate n/EVENT_NAME r/ROLES_NEEDED… d/DATE_AND_TIME l/LOCATION dsc/DESCRIPTION [m/MATERIALS_AND_LOGISTICS_NEEDED]... [b/BUDGET]` <br> e.g., `ecreate n/Fundraising at Orchard r/logistics leader d/23-09-2023 1500 dsc/station at locations to ask for donations m/50 tin cans b/50` |
+| **List all events**                           | `elist`                                                                                                                                                                                                                                                                                |
+| **Read an individual event**                  | `eshow EVENT_ID` <br> e.g., `eshow 8`                                                                                                                                                                                                                                                  |
+| **Delete an event**                           | `edelete EVENT_ID` <br> e.g., `edelete 3`                                                                                                                                                                                                                                              |
+| **Create a new volunteer profile**            | `vcreate vn/VOLUNTEER_NAME hp/PHONE_NUMBER e/EMAIL [s/SKILLS]...`<br> e.g.,`vcreate vn/John Lim hp/81234567 e/john123@gmail.com s/Cooking`                                                                                                                                             |
+| **List all volunteer profiles**               | `vlist`                                                                                                                                                                                                                                                                                |
+| **Delete a volunteer profile**                | `vdelete VOLUNTEER_ID` <br> e.g., `vdelete 4`                                                                                                                                                                                                                                          |
+| **Add a volunteer to an event**               | `eaddv vid/VOLUNTEER_ID /to eid/EVENT_ID`<br> e.g., `eaddv vid/1 /to eid/3` <br> Alternatively, `eaddv vn/VOLUNTEER_NAME /to en/EVENT_NAME` <br> e.g., `eaddv vn/Betsy Crowe /to en/Fundraising for needy families`                                                                    |
```

Comment:
Based on what we discussed earlier, i think the /to can be removed.

Reply:
dones